src/getCoinCombination.test.js

```diff
@@ -7,4 +7,63 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  describe('Currancy calculator', () => {
+    it('should return [0, 0, 0, 0] for 0 cents', () => {
+      expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+    });
+
+    it(`should return [1, 0, 0, 0] for 1 cents`, () => {
+      expect(getCoinCombination(1))
+        .toEqual([1, 0, 0, 0]);
+    });
+
+    it('should return [0, 1, 0, 0] for 5 cents', () => {
+      expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+    });
+
+    it(`should return [1, 1, 0, 0] for 6 cents`, () => {
+      expect(getCoinCombination(6))
+        .toEqual([1, 1, 0, 0]);
+    });
+
+    it(`should return [2, 1, 0, 0] for 7 cents`, () => {
+      expect(getCoinCombination(7))
+        .toEqual([2, 1, 0, 0]);
+    });
+
+    it('should return [3, 1, 0, 0] for 8 cents', () => {
```

Comment:
This test isn't needed as per the test design techniques) 
As per boundary values you should cover with inputs 1, 5, 10, 25+/- 1 and 1, 5, 10, 25 * 2 (If by this combination you will see duplicates - remove it)
Also add test which will return [1,1,1,1] as well